### PR TITLE
Update to Symfony 7

### PR DESCRIPTION
**Pull request by javiereguiluz**

### CODE REVIEW

File: tests/Command/AbstractCommandTest.php [hunk 1]
```diff
@@ -33,7 +34,9 @@ protected function executeCommand(array $arguments, array $inputs = []): Command
         // this uses a special testing container that allows you to fetch private services
         /** @var Command $command */
         $command = static::getContainer()->get($this->getCommandFqcn());
-        $command->setApplication(new Application(self::$kernel));
+        /** @var KernelInterface $kernel */
+        $kernel = self::$kernel;
```
Comment (GromNaN) [inline tests/Command/AbstractCommandTest.php:38]: Line 32, you can get the `$kernel = self::bootKernel();`.

File: composer.json [hunk 2]
```diff
@@ -13,37 +13,37 @@
         "symfony/polyfill-php81": "*"
     },
     "require": {
-        "php": ">=8.1",
+        "php": ">=8.2",
```
Comment (94noni) [inline composer.json:16]: what about php 8.3 ? (+ adding the polypill 8.2)

**Comment by javiereguiluz** [comment]
I don't understand the failing test. It says:

```
Verifying lock file contents can be installed on current platform.
Warning: The lock file is not up to date with the latest changes in composer.json. You may be getting outdated dependencies. It is recommended that you run `composer update` or `composer update <package name>`.
- Required package "symfony/phpunit-bridge" is not present in the lock file.
This usually happens when composer files are incorrectly merged or the composer.json file is manually edited.
Read more about correctly resolving merge conflicts https://getcomposer.org/doc/articles/resolving-merge-conflicts.md
and prefer using the "require" command over editing the composer.json file directly https://getcomposer.org/doc/03-cli.md#require-r
Error: Process completed with exit code 4.
```

BUT, `symfony/phpunit-bridge` is in the lock file:

https://github.com/symfony/demo/pull/1457/files#diff-f37acfaa6b11f575a9a6f41a75fa73a61d0f8ebc2c9b8cddc215d8aca10e44f5R8898

**Comment by dmaicher** [comment]
@javiereguiluz this seems related to https://github.com/sebastianbergmann/phpunit/issues/5578 and fixed by https://github.com/symfony/symfony/pull/52846

Maybe try using `composer require symfony/phpunit-bridge "7.0.x-dev as 7.0.1" --dev` to see if it works then

**Comment by javiereguiluz** [comment]
@dmaicher it worked! Thanks a lot 🙇

File: composer.json [hunk 3]
```diff
@@ -13,55 +13,55 @@
...
-        "symfony/css-selector": "6.4.*",
-        "symfony/debug-bundle": "6.4.*",
+        "symfony/browser-kit": "^7.0",
+        "symfony/css-selector": "^7.0",
+        "symfony/debug-bundle": "^7.0",
         "symfony/maker-bundle": "^1.36",
-        "symfony/phpunit-bridge": "6.4.*",
-        "symfony/stopwatch": "6.4.*",
-        "symfony/web-profiler-bundle": "6.4.*"
+        "symfony/phpunit-bridge": "7.0.x-dev as 7.0.1",
```
Comment (dmaicher) [inline composer.json:62]:
```suggestion
        "symfony/phpunit-bridge": "^7.0.1",
```

**Review by dmaicher — APPROVED** [review]
7.0.1 was relased :smiley: :+1:

File: composer.json [hunk 4]
```diff
@@ -13,55 +13,55 @@
         "symfony/polyfill-php81": "*"
     },
     "require": {
-        "php": ">=8.1",
+        "php": ">=8.2",
```
Comment (HeahDude) [inline composer.json:16]: The README needs an update of the requirements as well to reflect this change.

Comment (javiereguiluz) [reply composer.json:16]: Good catch! I've just updated this. Thanks.

**Review by GromNaN — APPROVED** [review]

**Pull request merged**